charts/seed-monitoring/charts/prometheus/templates/prometheus.yaml

```diff
@@ -61,6 +61,15 @@ spec:
     spec:
       # used to talk to Seed's API server.
       serviceAccountName: prometheus
+      initContainers:
+      - name: set-iptable-rules
+        image: alpine:3.8
```

Comment:
Don't hard-code this, we have it in our [`images.yaml`](https://github.com/gardener/gardener/blob/master/charts/images.yaml#L139-L141).